tetrateci/setup_boring_go.sh

```diff
@@ -14,11 +14,11 @@ if $(grep -q "1.10" <<< $TAG); then
   export GOLANG_VERSION=1.16.9b7
 fi
 
-if $(grep -q "1.11" <<< $TAG || grep -q "1.12" <<< $TAG); then
+if $(grep -q "1.11" <<< $TAG); then
   export GOLANG_VERSION=1.17.6b7
 fi
 
-if [[ "${REL_BRANCH_VER:-${ISTIO_MINOR_VER}}" == "1.13" ]]; then
+if $(grep -q "1.12" <<< $TAG) || [[ "${REL_BRANCH_VER:-${ISTIO_MINOR_VER}}" == "1.13" ]]; then
```

Comment:
this would make 1.12.0 - 1.12.5 starting use 1.17.8 as well. Previously they would use 1.17.6.
If we already release those images, then would cause difference.

If we only care about 1.12.6, then that's fine.

Reply:
I think that should be fine.We have already built 1.12.2 and plan is to build 1.12.6 patch release now.
cc @deva26

Reply:
We need to build 1.12.6 specifically with 1.17.8 for DoD as there scanner is picking up critical/Major CVE's related to 1.17.6 go version

Reply:
> and i dont think we will be building every patch release of istio unless it is specifically asked by customer or it contains significant code changes in term of CVE's or bug fixes.

ack. that's something I learned. make sense though.